sigma/plugins/roles/plugin.yml

```diff
@@ -49,6 +49,19 @@ commands:
       Makes a role self assignable.
       Requires the user who calls the command to have the Manage Roles permision.
 
+  - name: addselfcolorrole
+    alts:
+      - "ascr"
+      - "addselfcolourrole"
+    global:  true
+    enabled: true
+    sfw:     true
+    usage:   "{pfx:s}{cmd:s} <hex color> [name]"
+    description:
+      Makes a color role with the given color. If [name] is not given, it will use the color instead.
```

Comment:
The code doesn't seem to be using a 2nd argument. Is `name` functionality added?

Reply:
Right, I just patched that out. Let me fix that.